extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -83,6 +79,34 @@ export default function initWADOImageLoader(
   initWebWorkers(appConfig);
 }
 
+const generateAcceptHeader = (configAcceptHeader = [], omitQuotationForMultipartRequest = false) => {
```

Comment:
If you extract this function into a separate file, then it becomes easy to duplicate in two places - there isn't really a good solution for sharing it otherwise, but having it as a file makes it fairly easy.